apache-rat-plugin/pom.xml

```diff
@@ -195,25 +196,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
Did you intentionally remove the checkstyle config here? or is it a merging issue?